<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Detalles</class>
 <widget class="QMainWindow" name="Detalles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_logo_marca">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_marca">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>231</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_tipo">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>231</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_imagen">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>320</width>
      <height>420</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_modelo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>660</y>
      <width>410</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_precio">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>710</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>6</number>
    </property>
    <property name="indent">
     <number>4</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_volver">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>780</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Volver a la tabla de productos</string>
    </property>
    <property name="text">
     <string>VOLVER</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
